test/test_transforms_v2_refactored.py

```diff
@@ -1634,3 +1637,143 @@ def test_transform_negative_degrees_error(self):
     def test_transform_unknown_fill_error(self):
         with pytest.raises(TypeError, match="Got inappropriate fill arg"):
             transforms.RandomAffine(degrees=0, fill="fill")
+
+
+class TestCutMixMixUp:
+    # TODO: Does it work when labels are already dirichlet-distributed? like does Compose([Mixup(), CutMix()]) work?
+    @pytest.mark.parametrize("T", [transforms.Cutmix, transforms.Mixup, "Compose"])
+    @pytest.mark.parametrize("one_hot", [True, False])
+    def test_supported_input_structure(self, T, one_hot):
+
+        num_categories = 10
+        batch_size = 32
+        H, W = 12, 12
+
+        preproc = transforms.Compose([transforms.PILToTensor(), transforms.ToDtype(torch.float32)])
```

Comment:
Do we actually need this? If we go for the "reverse compose" as suggested above, each transform automatically gets passed probability based labels from the output of the transform that comes before.

Reply:
Sorry, I misplaced the comment. It should have been on the extra `ToOneHot` transform that we add below. If we have all four variants (both transforms individually and composed in both orders) we are sure that each one get a probably-based label at some point. Thus, we can drop the `one_hot` parametrization.

Reply:
As discussed offline I'm going to remove testing for Compose because this is probably mis-using the transforms. So we still need to have this `one_hot` parametrization